### PR TITLE
[ADD] pos_product_packaging_container_deposit: Introduce new module in 16.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**
Ref: BSALLO-607

### CODE REVIEW

File: pos_product_packaging_container_deposit/models/pos_order_line.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PosOrderLine(models.Model):
+    _inherit = "pos.order.line"
+
+    is_container_deposit = fields.Boolean()
```
Comment (santostelmo) [inline pos_product_packaging_container_deposit/models/pos_order_line.py:10]: The field is already present in `container.deposit.order.line.mixin`

Comment (nguyenminhchien) [reply pos_product_packaging_container_deposit/models/pos_order_line.py:10]: There is no field for the function `def _get_product_qty_delivered_received_field(self)`. So, i don't make pos.order.line inherits from that mixin model.

Comment (santostelmo) [reply pos_product_packaging_container_deposit/models/pos_order_line.py:10]: Indeed. Inherit mixin just to have the field `is_container_deposit` it's not worth it

**Comment by santostelmo** [comment]
Functional test done.  ✅

**Review by santostelmo — APPROVED** [review]

**Review by santostelmo — APPROVED** [review]
LG

**Review by simahawk — APPROVED** [review]
LG. Minor remark for the commit msg: `[ADD] pos_product_packaging_container_deposit` is enough :wink:

**Comment by nguyenminhchien** [comment]
> LG. Minor remark for the commit msg: `[ADD] pos_product_packaging_container_deposit` is enough 😉

i updated the commit message.

**Pull request merged**